content/documentation/getting-started/index.adoc

```diff
@@ -125,30 +132,64 @@ bash <(curl -L https://git.io/getLatestKialiOperator) --help
 icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
 
 
-=== Install or Update Kiali
+=== Create or Edit the Kiali CR
 
-The Kiali Operator watches the Kiali CR (Custom Resource).  Create, update, or removal of the Kiali CR will trigger the Kiali Operator to install, update, or remove Kiali.
+The Kiali Operator watches the Kiali CR (Custom Resource).  Create, update, or removal of the Kiali CR will trigger the Kiali Operator to install, update, or remove Kiali. This assumes the Kiali Operator has already been installed.  See link:#_advanced_install_operator_only[Advanced Install (Operator-Only)] if you need to install the Kiali Operator. 
 
-If you performed the link:#_operator_only_install[Operator-Only Install] you must then generate a Kiali CR file to install Kiali. It is recommended to copy the fully documented link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[example Kiali CR yaml file].  Edit the file, being careful to maintain proper formatting, and save it to a local file (e.g. my-kiali-cr.yaml).
+To create an initial Kiali CR file it is recommended to copy the fully documented link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[example Kiali CR yaml file].  Edit the file, being careful to maintain proper formatting, and save it to a local file (e.g. my-kiali-cr.yaml).
 
 icon:lightbulb[size=1x]{nbsp} It is important to understand the `deployment:accessible_namespaces` setting in the CR.  See link:#_accessible_namespaces[Accessible Namespaces] for more information.
 
-To create (or update) the Kiali CR using a local file, run the command:
+To install Kiali create the Kiali CR using the local file. To create the Kiali CR run the command:
 
 [source,bash]
 ----
 oc apply -f my-kiali-cr.yaml -n kiali-operator
 ----
 
-
-You can also update Kiali by editing the Kiali CR.  To edit the Kiali CR run the command (note: the default Kiali CR name is `kiali`):
+To update Kiali edit and save the existing the Kiali CR.  To edit the Kiali CR run the command (note: the default Kiali CR name is `kiali`):
 
 [source,bash]
 ----
 oc edit kiali <Kiali CR name> -n kiali-operator
 ----
 
 
+== Uninstall
+
+=== Uninstall Kiali Only
+
+To uninstall Kiali is simple - just delete the Kiali CR. To trigger the Kiali Operator to uninstall Kiali run the command (note: the default Kiali CR name is `kiali`):
+
+[source,bash]
+----
+oc delete kiali <Kiali CR name> -n kiali-operator
+----
+
+At this point, you have no Kiali installed, but you still have the Kiali Operator running. You could create another Kiali CR (with potentially different configuration settings) to install a new Kiali instance.
+
+=== Wipe Kiali
```

Comment:
Word choice - "wipe" might be confusing, especially for ESL.  I know we need to distinguish between uninstalling Kiali and uninstalling Kiali + Kiali operator, and I'm not sure if "uninstall" and "remove" are distinct enough, but how about we go with that until someone comes up with a better idea?

Reply:
+1, see my response above